jira/resources.py

```diff
@@ -358,7 +329,8 @@ def update(
         r = self._session.put(self.self + querystring, data=json.dumps(data))
         if "autofix" in self._options and r.status_code == 400:
             user = None
-            error_list = get_error_list(r)
+            raise_on_error(r)
+            error_list = parse_errors(r)
             logging.error(error_list)
```

Comment:
would we want to log the error and afterwards raise?

Reply:
it looks like raising the error should be removed completely as it will always raise for 400 HTTP response
previously it was raised in the `json_loads` and caught by `get_error_list` code